app/views/schools/users/index.html.erb

```diff
@@ -21,13 +21,13 @@
             <td><%= user.email %></td>
             <td><%= y_n(user.confirmed?) %></td>
             <td><%= y_n(user.contact_for_school) %></td>
```

Comment:
Wondering whether the "y_n" function should have a translated version.. what do you think, have we had the same elsewhere?

Reply:
hi @jhigman translation support to the y_n method was in a pr merged earlier in the week while you were away so should be all ok now :)  it looks like this now 
```
  def y_n(boolean)
    boolean ? I18n.t('common.labels.yes_label') : I18n.t('common.labels.no_label')
  end 
```